ports/servoshell/egl/app.rs

```diff
@@ -332,21 +332,19 @@ impl App {
         &self.state.servoshell_preferences
     }
 
-    pub(crate) fn focused_or_newest_webview(&self) -> Option<WebView> {
-        self.state.any_window().focused_or_newest_webview()
+    pub(crate) fn active_or_newest_webview(&self) -> Option<WebView> {
+        self.state.any_window().active_or_newest_webview()
     }
 
-    pub(crate) fn create_and_focus_toplevel_webview(self: &Rc<Self>, url: Url) -> WebView {
+    pub(crate) fn create_and_activate_toplevel_webview(self: &Rc<Self>, url: Url) -> WebView {
         self.state
             .any_window()
-            .create_and_focus_toplevel_webview(self.state.clone(), url)
+            .create_and_activate_toplevel_webview(self.state.clone(), url)
     }
 
-    /// The focused webview will not be immediately valid via `focused_or_newest_webview()`
-    pub(crate) fn focus_webview(&self, id: WebViewId) {
-        if let Some(webview) = self.state.webview_by_id(id) {
-            webview.focus();
-        }
+    /// The active webview will be immediately valid via `active_or_newest_webview()`
```

Comment:
Is the change in the semantics here intentional? I believe previous comment was highlighting the fact that focusing a webview is asynchronous.